data/json/mutations/mutation_spells.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "cephalopod_ink_spray",
+    "type": "SPELL",
+    "name": "Ink spray",
+    "description": "Blind enemies for a short time with an ink gland discharge.",
+    "effect": "cone_attack",
+    "effect_str": "blind",
+    "affected_body_parts": [ "EYES" ],
+    "valid_targets": [ "hostile" ],
+    "max_level": 5,
+    "min_damage": 0,
+    "max_damage": 1,
+    "damage_increment": 0.0,
+    "min_aoe": 0,
```

Comment:
no need to write the default values, that's redundant. If the damage_increment is 0, then you will never get from the minimum of 0 to the maximum of 1. it is reasonable to post a minimum value of 0 if there is a max value of nonzero though.